### DISCUSSION TITLE
Paperless-ng setup?

### DISCUSSION THREAD

**Discussion by rdrr** [NginxProxyManager/nginx-proxy-manager#1833] [Q&A]
Can anyone help me set this up?  I though I understood how to create a location, but whenever the 'proxy_pass http://IP or localhost:8000' is added to it, the status goes offline and when I navigate to the url, it's no longer serving the Let's Encrypt certificate, but a dummy cert issued by NPM.  Leaving off the proxy_pass give me a 520.

**Comment by mochman** [comment]
I have npm running in front of paperless-ng on docker.  Is that your setup too?  I didn't have to add anything to the Custom Locations tab in order to get https working.  How do you have yours configured?

**Comment by Maximus48p** [comment]
I copied the example from Paperless-ngx website, but without succes. 
Does anyone has a working example with the latest version?
thx!
